pkg/pipeline/provider.go

```diff
@@ -41,22 +41,19 @@ func Provider() *schema.Provider {
 			},
 		},
 
-		ResourcesMap: util.AddTelemetry(
-			productId,
-			map[string]*schema.Resource{
-				"pipeline_source":              pipelineSourceResource(),
-				"pipeline_project_integration": pipelineProjectIntegrationResource(),
```

Comment:
We need to keep this resource and add deprecation message to it. After this initial release we can then gradually phase out this resource.

See [Terraform recommendation](https://www.terraform.io/plugin/sdkv2/best-practices/deprecations#provider-data-source-or-resource-removal)

Reply:
Generic integration resource added back in.